dex-load/src/main/scala/com/wavesplatform/dex/load/WavesDexLoadCli.scala

```diff
@@ -346,8 +346,11 @@ object WavesDexLoadCli extends ScoptImplicits {
   // The compiler is lie! This is used in WsOrder.id
   @silent("never used") private implicit val derivedByteStrDiff: Derived[Diff[ByteStr]] = Derived(getDiff[ByteStr](_.toString == _.toString))
 
-  private implicit val wsAddressChangesDiff: Diff[WsAddressChanges]     = Derived[Diff[WsAddressChanges]].ignore(_.timestamp).ignore(_.updateId)
-  private implicit val wsOrderBookChangesDiff: Diff[WsOrderBookChanges] = Derived[Diff[WsOrderBookChanges]].ignore(_.timestamp).ignore(_.updateId)
+  private implicit val wsAddressChangesDiff: Diff[WsAddressChanges] =
+    Derived[Diff[WsAddressChanges]].ignore[WsAddressChanges, Long](_.timestamp).ignore[WsAddressChanges, Long](_.updateId)
```

Comment:
Interesting, is this really required?
```scala
[WsAddressChanges, Long]
```

Reply:
Yep, otherwise we got `missing parameter type for expanded function`